strings/ion_laws.json

```diff
@@ -1042,5 +1042,16 @@
         "SPYING ON",
         "STALKING",
         "WATCHING"
-    ]
+    ],
+	"ionpet": [
+		"POLY",
+		"RENAULT",
+		"IAN",
+		"PUN PUN",
+		"LAMARR",
+		"RUNTIME",
+		"CITRUS",
+		"MCGRIFF",
+		"ARANEUS"
```

Comment:
I don’t really like this. Pets change or aren’t present on every map. Could you look through the mob list insteaf and pick pets from that?

Reply:
The purpose of the laws is to explicitly require the AI to care about station pets, not to get them to care about animals in general. I know the laws have a chance of just not doing anything in contexts where those pets don't exist on that map; if we removed every ion law that had a chance of not working because of the current station state, we would have maybe 5 of them.

Reply:
Imo that’s preferable and I disagree but also w/e we ball